manifests/server.pp

```diff
@@ -262,6 +262,9 @@
 #
 # $connect_timeout::           How long the server will wait for a response to a connection attempt
 #
+# $server_web_idle_timeout::   Time in ms that Jetty allows a socket to be idle, after processing has completed.
```

Comment:
This parameter isn't added to the class. Looks like this filename is not in the docs checks that would verify this.